tests/driver_ds3231/main.c

```diff
@@ -94,7 +94,12 @@ static int _cmd_get(int argc, char **argv)
     char dstr[ISOSTR_LEN];
 
     struct tm time;
-    ds3231_get_time(&_dev, &time);
+    int res;
+    res = ds3231_get_time(&_dev, &time);
+    if (res != 0) {
+        puts("error: unable to read time");
```

Comment:
```suggestion
        DEBUG("error: unable to read time\n");
```

Reply:
It is ok to use puts in a test application code.

Reply:
ah I didn't realize this was the test, not the driver